pkg/convert/pot_test.go

```diff
@@ -0,0 +1,298 @@
+package convert
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+func Test_ToPot(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name     string
+		input    model.Messages
+		expected []byte
+	}{
+		{
+			name: "When all values are provided",
+			input: model.Messages{
+				Language: language.English,
+				Messages: []model.Message{
+					{
+						ID:          "Hello, world!",
+						Message:     "Bonjour le monde!",
+						Description: "A simple greeting",
+						Fuzzy:       true,
+					},
+					{
+						ID:          "Goodbye!",
+						Message:     "Au revoir!",
+						Description: "A farewell",
+						Fuzzy:       true,
+					},
+				},
+			},
+			expected: []byte(
+				"\"Language: en\n" +
+					"#. A simple greeting\n#, fuzzy\nmsgid \"Hello, world!\"\nmsgstr \"Bonjour le monde!\"\n" +
+					"#. A farewell\n#, fuzzy\nmsgid \"Goodbye!\"\nmsgstr \"Au revoir!\"\n"),
+		},
+		{
+			name: "When msgid is multiline",
+			input: model.Messages{
+				Language: language.English,
+				Messages: []model.Message{
+					{
+						ID:          "Hello, world!\\n\"\n\"very long string\\n",
+						Message:     "Bonjour le monde!",
+						Description: "A simple greeting",
+						Fuzzy:       true,
+					},
+					{
+						ID:          "Goodbye!",
+						Message:     "Au revoir!",
+						Description: "A farewell",
+						Fuzzy:       true,
+					},
+				},
+			},
+			expected: []byte(
+				"\"Language: en\n" +
+					"#. A simple greeting\n#, " +
+					"fuzzy\n" +
+					"msgid \"\" \n\"Hello, world!\\n\"\n\"very long string\\n\"\n" +
+					"msgstr \"Bonjour le monde!\"\n" +
+					"#. A farewell\n" +
+					"#, fuzzy\nmsgid \"Goodbye!\"\n" +
+					"msgstr \"Au revoir!\"\n"),
```

Comment:
To improve readability, I suggest using multiline strings, which are much easier to read.

```suggestion
			expected: []byte(`Language: en
#. A simple greeting
#, fuzzy
msgid ""
....`),
```